crates/wash-cli/tests/fixtures/integrated-wkg-extended/wasmcloud.toml

```diff
@@ -0,0 +1,22 @@
+name = "ponger-config-component"
+language = "rust"
+type = "component"
+
+[component]
+wasm_target = "wasm32-wasip2"
+
+[[registry.pull.sources]]
+target = "wasmcloud:bus"
+source = "oci://ghcr.io/wasmcloud/interfaces"
+
+[[registry.pull.sources]]
+target = "test-components:testing"
+source = "file://extra-wit/pingpong.wit"
+
+[[registry.pull.sources]]
+target = "wasi:config"
+source = "https://github.com/WebAssembly/wasi-config/archive/v0.2.0-draft.tar.gz"
+
+[[registry.pull.sources]]
+target = "wasi:blobstore"
+source = "git+https://github.com/WebAssembly/wasi-blobstore.git"
```

Comment:
For testing, we should probably make sure this is going to pull from a pinned SHA to avoid any issues down the line

Reply:
The way this works is that it actually runs `git clone` -- so is the idea here to add a `?sha=<....>`  or to have some sort of URI encoding that exposes a SHA we should set ourselves to later?

In the case that you want a specific point of a git repo I think you'd be trying to use a regular `https://` target (like we do above for the `wasi-config` at `v0.2.0-draft` tag...

Maybe I'm misunderstanding, what's the code that you would replace this line with?

Reply:
Yeah, what will happen here with the test if we don't checkout a specific sha is that we could make a change on `main` and then the test will fail because the wit changed. Ideally, when we fold this into main `wkg`, the git sha it picks on clone would be written to the lock file. But right now, this will always pull from the latest main, which is probably not desired behavior. Adding the `sha` param was the solution I was thinking of (which can be a commit sha or a tag or a branch name).

Reply:
Done -- make sure to add this, and made it a bit more explicit